packages/tezt/tezt.1.0.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "2.0" }
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
  "dune" { >= "2.0" }
```

Reply:
Thanks, I came to the same conclusion.

Reply:
It may also need a lower bound on dune. The lower bound tests, downgrading to dune 2.0.1 fails at https://gitlab.com/tezos/tezos/-/blob/master/src/lib_micheline/test/dune#L10 with
```
#=== ERROR while compiling tezt.1.0.0 =========================================#
# context              2.1.0~rc2 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://gitlab.com/tezos/tezos/-/archive/v10.0-rc1/tezos-v10.0-rc1.tar.bz2)
# path                 ~/.opam/4.08/.opam-switch/build/tezt.1.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p tezt -j 47
# exit-code            1
# env-file             ~/.opam/log/tezt-6473-4bfab6.env
# output-file          ~/.opam/log/tezt-6473-4bfab6.out
### output ###
# File "src/lib_micheline/test/dune", line 10, characters 2-16:
# 10 |   (action progn))
#        ^^^^^^^^^^^^^^
# Error: if you meant for this to be executed with bash, write (bash "...")
# instead
```

Reply:
We might want to disable tests from other Tezos modules.

Reply:
In fact, there's a missing `(package ...)` in the dune file, I'll open a merge request on Tezos first.